sqlite3.h

```diff
@@ -1,3 +1,40 @@
+/*
```

Comment:
Similar to the C files we'll have to somehow use build tags to control which header file to use.

Reply:
Possibly need your help on this